dist/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Odin Restaurant</title>
+    <link href="css/main.css" rel="stylesheet" type="text/css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700,900&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <div id="content">
+        <nav>
```

Comment:
I didn't expect to see any code here other than an empty `<div id="content"> </div>` you will use JavaScript in your `index.js` file to append views to the `div`. All the coding will be done in your src folder and `webpack`  after runing `npm run build` will compile it and generate it into the `main.js` in your `dist`.